ports/renesas-ra/Makefile

```diff
@@ -171,6 +175,26 @@ SHARED_SRC_C += $(addprefix shared/,\
 	runtime/stdout_helpers.c \
 	runtime/sys_stdio_mphal.c \
 	timeutils/timeutils.c \
+	tinyusb/mp_cdc_common.c \
+	tinyusb/mp_usbd.c \
+	tinyusb/mp_usbd_descriptor.c \
+	)
+
+# TinyUSB Stack source
+SRC_TINYUSB_C += $(addprefix lib/tinyusb/,\
+	src/class/cdc/cdc_device.c \
+	src/class/dfu/dfu_rt_device.c \
+	src/class/hid/hid_device.c \
+	src/class/midi/midi_device.c \
+	src/class/msc/msc_device.c \
+	src/class/usbtmc/usbtmc_device.c \
+	src/class/vendor/vendor_device.c \
+	src/common/tusb_fifo.c \
+	src/device/usbd.c \
+	src/device/usbd_control.c \
+	src/portable/renesas/rusb2/dcd_rusb2.c \
+	src/portable/renesas/rusb2/hcd_rusb2.c \
```

Comment:
```suggestion
	src/portable/renesas/rusb2/hcd_rusb2.c \
	src/portable/renesas/rusb2/rusb2_common.c \
```

Reply:
Is the suggestion here an extra indentation ?

Reply:
Ah I see those are different files, well the TinyUSB is still a work in progress upstream, I keep updating this and retesting, it's not done yet but almost ready.

Reply:
No worries, I just added it as I saw I needed it (as well as the updated tinyusb submodule from the PR) in order to build.